db/test-setup.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
+#!/bin/bash
```

Comment:
ouch! nice one

Reply:
First three commits on this PR are from https://github.com/taskcluster/taskcluster/pull/5496 btw :)

Reply:
hmm, actually space is allowed :D https://en.wikipedia.org/wiki/Shebang_(Unix)#History  who knew

Reply:
> First three commits on this PR are from #5496 btw :)

Approving both